src/activity_handlers.cpp

```diff
@@ -41,6 +41,7 @@
 #include "mongroup.h"
 #include "morale_types.h"
 #include "mtype.h"
+#include "npc.h"
```

Comment:
Don't see where this is used, but I could be wrong.
```suggestion
```

Reply:
Think that was a leftover for when I was trying to do the NPC skill training in the activity, something wasnt found, but i think its ok now.